### PR TITLE
fix #70031 babesinporn.com

### LINKED ISSUE #70031: babesinporn.com
### Issue URL (Ads)
[NSFW] [https://www.babesinporn.com/kimber-cox/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.babesinporn.com%2Fkimber-cox%2F)

### [NSFW] Screenshots
<details>
  <summary>Screenshot 1</summary>

  [Screenshot 1](https://reports-img.adguard.com/iyrYRy6.jpg)
</details>
<details>
  <summary>Screenshot 2</summary>

  [Screenshot 2](https://reports-img.adguard.com/3SlPJO4.jpg)
</details>

### System configuration

Information | value
--- | ---
Platform: | uBlock Origin
Browser: | Firefox
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base, <br/>EasyList<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances

### PR DESCRIPTION
**Pull request by stanislav-atr**
#70031

### CODE REVIEW

File: EnglishFilter/sections/specific.txt [hunk 1]
```diff
@@ -264,6 +264,9 @@ lookmovie.io##.bottom-message-container
 lookmovie.io##div[class$="-page-ads-wrapper"]
 lookmovie.io##.notifyjs-corner
 pixiv.net##a[target="premium_noads"]
+||babesinporn.com/flr.js
+babesinporn.com##.list > div.item > a[target="_parent"] > img[width="240"]
+babesinporn.com##body > div[style^="display: block; position: fixed; z-index:"]
```
Comment (zloyden) [inline EnglishFilter/sections/specific.txt:269]:
```
||content*.babesinporn.com/images/*/istripper/
||content*.babesinporn.com/images/*/strp_v*.js
```
You can block requests, also it's better for second rule. You have also missed the `playboy` ad. `||content*.babesinporn.com/images/*/banners/`

**Review by zloyden — CHANGES_REQUESTED** [review]

**Review by zloyden — APPROVED** [review]

File: EnglishFilter/sections/specific.txt (hunk 1 above)
Comment (Yuki2718) [reply EnglishFilter/sections/specific.txt]: @zloyden that istripper and banners are shared with many similar sites. I've added a common rule to uAssets recently, will open PR.

Comment (zloyden) [reply EnglishFilter/sections/specific.txt]: I have noticed it but we don't make these rules `generic` right away.
For now there are common rules with domains.

Comment (Yuki2718) [reply EnglishFilter/sections/specific.txt]: I agree not to make generic. As you see in my PR, just adding domains. Apparently flr.js is not common, but I'll add few more domains to banners. OTOH I'm not sure about /bload, it seems usually not loaded. You can find inline script to load `/bloader` conditionally in e.g. `https://www.rossoporn.com/grace-beautiful-nude-girl`:

**Review by ShadowsatNight — COMMENTED** [review]
Toast!

**Pull request merged**